src/sql/workbench/contrib/dashboard/browser/contents/dashboardWidgetWrapper.css

```diff
@@ -40,17 +40,18 @@ dashboard-widget-wrapper .noTitle {
 }
 
 dashboard-widget-wrapper .actionbar {
-	flex: 0 0 auto;
 	align-self: end;
 }
 
 dashboard-widget-wrapper .bottomActionbar {
-	flex: 0 0 auto;
 	align-self: center;
-	margin-top: -27px;
 	display: none;
 }
 
+dashboard-widget-wrapper .bottomActionbar.collapsed {
+	margin-top: -27px;
```

Comment:
I wanted to get rid of this too but this was a lot trickier due to how we have it set up using the actionbar as the bottom bar on the same line as the widget title. I at least made it so this is only necessary when it's collapsed and hopefully I'll be able to come back and fix this up better in the future.